users/id-b3/README.md

```diff
@@ -0,0 +1,3 @@
+User configuration for id-b3
```

Comment:
As above.

Reply:
The folder name is linked to my github username, and to the userspace. I see other users with - in their folder names and keymaps on the main repo.